server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala

```diff
@@ -143,7 +143,7 @@ class AddressIndexRepository @Inject()(conf: ConfigModule,
     val boostExponent = 1.2
 
     val fromSourceQueryShould =
-      if (eboost == 1 && sboost == 1 && nboost == 1 & wboost ==1) Seq.empty
+      if ((eboost == 1 || eboost == 0) && (sboost == 1 || sboost == 0) && (nboost == 1 || nboost == 0) & (wboost == 0 || wboost ==1)) Seq.empty
```

Comment:
should there be a double ampersand instead of single?

Reply:
well spotted